### PR TITLE
CM-763: How to Target Multiple Frameworks in a Single .NET Project

### PR DESCRIPTION
**Pull request by DharaVaghasiya**
Added Example project for CM-763- How to Target Multiple Frameworks in a Single .NET Project

### CODE REVIEW

File: dotnet-projects/MultipleFrameworkTarget/MultipleFrameworkTargetConsole.sln [hunk 1]
```diff
@@ -0,0 +1,31 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
+VisualStudioVersion = 17.6.33829.357
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "MultipleFrameworkTargetConsole", "..\..\..\..\..\MultipleFrameworkTargetConsole\MultipleFrameworkTargetConsole.csproj", "{1291FFE2-345C-4576-B242-D670EC9C8DA3}"
```
Comment (jshergal) [inline dotnet-projects/MultipleFrameworkTarget/MultipleFrameworkTargetConsole.sln:6]: Something is messed up in this solution file and that is causing the build to fail. If you look at the path that the solution has for the MultipleFrameworkTargetConsole.csproj, it is looking for it five directories up from the solution location

Comment (fl0tschi) [reply dotnet-projects/MultipleFrameworkTarget/MultipleFrameworkTargetConsole.sln:6]: @DharaVaghasiya please have a look at this what @jshergal pointed out. This looks definitely like the problem :-)

File: dotnet-projects/MultipleFrameworkTarget/Tests/Tests.csproj [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.5.0" />
+    <PackageReference Include="MSTest.TestAdapter" Version="2.2.10" />
+    <PackageReference Include="MSTest.TestFramework" Version="2.2.10" />
+    <PackageReference Include="coverlet.collector" Version="3.2.0" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\..\..\..\MultipleFrameworkTargetConsole\MultipleFrameworkTargetConsole.csproj" />
```
Comment (fl0tschi) [inline dotnet-projects/MultipleFrameworkTarget/Tests/Tests.csproj:20]: Also here the project reference looks wrong. Should be max. 1 directory up.

**Review by fl0tschi — CHANGES_REQUESTED** [review]

**Comment by fl0tschi** [comment]
retest this please

**Review by fl0tschi — APPROVED** [review]

**Pull request merged**